src/components/Menus/Sidebar/ImagePanel/ProjectImages/ProjectImages.scss

```diff
@@ -2,25 +2,26 @@
 @import "../../../../../colours.scss";
 
 .project-images {
-  text-align: center;
+  display: grid;
+  grid-template-columns: repeat(auto-fit, $space-7-5);
+  justify-content: center;
+  gap: $space-1;
 
   &__block {
     display: inline-block;
     box-sizing: border-box;
-    padding-left: $space-0-5;
     overflow-wrap: break-word;
     word-break: break-word;
     text-align: center;
     vertical-align: top;
   }
 
   &__image-wrapper {
-    height: $space-5-5;
-    width: $space-5-5;
+    height: $space-7-5;
+    width: $space-7-5;
```

Comment:
In the [Design](https://www.figma.com/file/Rbn27JvH8A8wwqagKSReIq/Code-Editor---Component-library?node-id=1150%3A13119&mode=dev), it looks like the single-column images are slightly larger than the two-column ones.  

Perhaps worth flagging up with Mac to confirm?

Reply:
Yeah in the design they have the images resizing when you change the width of the sidebar, but felt this was overkill and if you have the images growing as you widen the sidebar it kind of defeats the object of widening it I think because if the images grow you get to see fewer images, not more 🤔